### PR TITLE
Doc install

### PR DESCRIPTION
**Pull request by sphamba**
# Prework

- [x] I understand and agree to this repository's [code of conduct](https://github.com/ghiggi/gpm_api/blob/main/CODE_OF_CONDUCT.md).
- [x] I understand and agree to this repository's [contributing guidelines](https://github.com/ghiggi/gpm_api/blob/main/CONTRIBUTING.md).
- [x] I have already submitted an [issue](https://github.com/ghiggi/gpm_api/issues) or [discussion thread](https://github.com/ghiggi/gpm_api/discussions) to discuss my idea with the maintainers.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Tutorial
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and communicate accordingly:

**The PR fulfills these requirements:**

- [ ] It's submitted to the branch named as follow :
  - Fix a bug: `bugfix-<some_key>-<word>`
  - Improve the doc: `doc-<some_key>-<word>`
  - Improve a tutorial `tutorial-<some_key>-<word>`
  - Add a new feature: `feature-<some_key>-<word>`
  - Refactor some code: `refactor-<some_key>-<word>`
  - Optimize some code: `optimize-<some_key>-<word>`
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] Don't forget to link PR to issue if you are solving one.
- [ ] All tests are passing.
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

# Related GitHub issues and pull requests

- Ref: #

# Summary

Added installation instructions (with details for Cartopy)
[Readthedocs](https://gpm-api--10.org.readthedocs.build/en/10/installation.html#installation-for-standard-users)

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -1,5 +1,170 @@
...
+    git clone https://github.com/<your-account>/gpm_api.git
+    cd gpm_api
+
+
+
+To install the project in editable mode :
+
+.. code-block:: bash
+
+	pip install -e .[image,dev]
```
Comment (evanjt) [inline docs/source/installation.rst:82]: Does this work for you? I need to do a `pip install -e ".[image,dev]"` for local paths

**Review by evanjt — APPROVED** [review]

File: docs/source/installation.rst (hunk 1 above)
Comment (sphamba) [reply docs/source/installation.rst:82]: Oh, it does work for me without the quotes. I'll still add them

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Gionata
+    family-names: Ghiggi
+    email: gionata.ghiggi@epfl.ch
+    affiliation: EFPL
+    orcid: 'https://orcid.org/0000-0002-0818-0865'
+  - given-names: Randy
```
Comment (ghiggi) [inline CITATION.cff:13]:
```suggestion
```

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    family-names: Chase
```
Comment (ghiggi) [inline CITATION.cff:14]:
```suggestion
```

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    email: randy.chase12@gmail.com
```
Comment (ghiggi) [inline CITATION.cff:15]:
```suggestion
```

**Review by ghiggi — APPROVED** [review]
LGTM

**Pull request merged**